.github/workflows/node.js.yml

```diff
@@ -0,0 +1,29 @@
+# This workflow will do a clean install of node dependencies, build the source code and run tests across different versions of node
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+
+name: Node.js CI
+
+on:
+  push:
+    branches: [ develop ]
```

Comment:
Should we configure the `deployment` for push on `master` branch too?

Reply:
for deployment, my initial plan is to deploy from `develop`, gradually we'll deploy release from `master`